api/src/organizational-entities/domain/models/Organization.js

```diff
@@ -58,36 +58,40 @@ class Organization {
     this.archivedAt = archivedAt;
   }
 
-  get isSup() {
-    return this.type === types.SUP;
+  get isAgriculture() {
+    return Boolean(this.tags.find((tag) => this.isSco && tag.name === Tag.AGRICULTURE));
   }
 
-  get isSco() {
-    return this.type === types.SCO;
+  get isArchived() {
+    return this.archivedAt !== null;
+  }
+
+  get isGarIdentityProvider() {
```

Comment:
En soi l'*Organization* n'est pas un *GarIdentityProvider*, mais peut en avoir un, c'est pourquoi il me semble que l'utilisation de `has` plutôt que `is` serait plus adaptée : 

```suggestion
  get hasGarIdentityProvider() {
```